spec/models/workflow_object_spec.rb

```diff
@@ -15,17 +15,18 @@
     end
   end
 
+  # TODO: Move to the DataIndexer spec
```

Comment:
Does this TODO need to be done?

Reply:
It was not my intention to do it in this PR